src/main/java/mate/academy/service/impl/CinemaHallServiceImpl.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.service.impl;
+
+import java.util.List;
+import mate.academy.dao.CinemaHallDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.CinemaHall;
+import mate.academy.service.CinemaHallService;
+
+@Service
+public class CinemaHallServiceImpl implements CinemaHallService {
+    @Inject
+    private CinemaHallDao cinemaHallDao;
+
+    @Override
+    public CinemaHall add(CinemaHall cinemaHall) {
+        return cinemaHallDao.add(cinemaHall);
+    }
+
+    @Override
+    public CinemaHall get(Long id) {
+        return cinemaHallDao.get(id).orElseThrow(() -> new DataProcessingException(
+                "Can not get CinemaHall by id: " + id));
```

Comment:
DataProcessingException means we have problems with database and we cannot retrieve/update it.
Here everything is fine with db, we just don't have required info saved there.
Please use proper Exception type. NoSuchElement, EntityNotFound or any other relevant for the case
```suggestion
        return cinemaHallDao.get(id).orElseThrow(() -> new NoSuchElementException(
                "Can not get CinemaHall by id: " + id));
```